dataplane/src/packet_processor/ipforward.rs

```diff
@@ -80,19 +80,19 @@ impl IpForwarder {
 
         /* Read-only access to the fib table */
         let Some(fibtr) = self.fibtr.enter() else {
-            error!("{nfi}: Unable to lookup fib for vrf {vrfid}");
```

Comment:
this can likely remain an `error`.  If the user is able to give us messages for vrfs which don't exist then we have a more fundamental problem than DoS

Reply:
I think warn is fair

Reply:
Fine with me!